compass/logging.py

```diff
@@ -64,3 +63,29 @@ def log_method_call(method, logger):
         logger.info(f'  inherited from: {actual_location}.{method_name}()')
     if class_file is not None:
         logger.info(f'  in {class_file}')
+
+
+def log_function_call(function, logger):
+    """
+    Log the module path and file path of a call to a function, e.g.::
+
+      compass calling: compass.parallel.set_cores_per_node()
+        in /home/xylar/code/compass/compass/compass/parallel.py
+
+    Parameters
+    ----------
+    function : function
+        The function that will be run immediately following this call
+
+    logger: logging.Logger
+        The logger to log the method path and file path to
```

Comment:
This should be `The logger to log the function path and file path to`

Reply:
Done!